src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py

```diff
@@ -58,22 +58,37 @@ class SupportedImages(Enum):  # pylint: disable=too-few-public-methods
                help='Relative mount path for nfs. The nfs will be available at '
                     '$AZ_BATCHAI_MOUNT_ROOT/<relative_mount_path> folder.',
                arg_group='File Server Mount')
+    c.argument('account_name', options_list=('--account-name',),
+               help='Storage account name for Azure File Shares and/or Azure Storage Containers mounting. Related '
+                    'environment variable: AZURE_BATCHAI_STORAGE_ACCOUNT. Must be used in conjunction with '
+                    'storage-account-key. If the key is not provided, the command will try to query the storage '
+                    'account key using the authenticated Azure account.',
+               arg_group='Storage Account')
+    c.argument('account_name', options_list=('--account-name',),
+               help='Storage account name for Azure File Shares and/or Azure Storage Containers mounting. Related '
+                    'environment variable: AZURE_BATCHAI_STORAGE_ACCOUNT. Must be used in conjunction with '
+                    '--account-key. If the key is not provided, the command will try to query the storage '
+                    'account key using the authenticated Azure account.',
+               arg_group='Storage Account')
```

Comment:
This looks like a duplicate of the above?

Reply:
thanks, removed